test/cpp/ext/csm/metadata_exchange_test.cc

```diff
@@ -243,6 +243,25 @@ class MetadataExchangeTest
     EXPECT_EQ(attributes.find("csm.remote_workload_type"), attributes.end());
   }
 
+  // Returns the expected size of attributes on client/server metrics (other
+  // than `grpc.client.attempt.started` and `grpc.server.call.started`). Use of
+  // this method additionally tests `Size()` method on `MeshLabelsIterable`
+  // which is unused by the OTel C++ library at present, but might be used in
+  // the future.
+  size_t ExpectedAttributesSize(bool is_client) {
```

Comment:
Instead of integrating this into this test, how about adding a separate unit test for `MeshLabelsIterable` directly, since that's actually the interface we're trying to test in this case?